app/src/components/menu/menu.js

```diff
@@ -5,16 +5,21 @@ import {Menu, shell, clipboard} from 'electron';
  * @param appQuit
  * @param zoomIn
  * @param zoomOut
+ * @param zoomReset
+ * @param zoomBuildTimeValue
  * @param goBack
  * @param goForward
  * @param getCurrentUrl
  * @param clearAppData
  * @param disableDevTools
  */
-function createMenu({nativefierVersion, appQuit, zoomIn, zoomOut, goBack, goForward, getCurrentUrl, clearAppData, disableDevTools}) {
+function createMenu({nativefierVersion, appQuit, zoomIn, zoomOut, zoomReset, zoomBuildTimeValue, goBack, goForward, getCurrentUrl, clearAppData, disableDevTools}) {
     if (Menu.getApplicationMenu()) {
         return;
     }
+    const zoomResetLabel = (zoomBuildTimeValue === 1.0) ?
+      'Zoom Reset' :
```

Comment:
@jiahaog I put it this way to have "Zoom things" (In, Out, Reset) visually together, but after all they're already together given the menu position and grouping. Will change.